mech_client/marketplace_interact.py

```diff
@@ -90,9 +116,9 @@ def send_marketplace_request(  # pylint: disable=too-many-arguments,too-many-loc
     method_name = "request"
     methord_args = {
         "data": v1_file_hash_hex_truncated,
-        "priorityMech": ADDRESS_ZERO,
+        "priorityMech": "0x2C347caF85475793A19B18925faA3BEbEa27cd5c",
```

Comment:
We want this to be configurable

Reply:
The default value should be a global var like `ADDRESS_ZERO`, makes it more readable, and easier to change later if need be.